README.md

```diff
@@ -14,7 +14,7 @@ wasi-i2c is currently in [Phase 1](https://github.com/WebAssembly/WASI/blob/main
 
 ### Portability Criteria
 
-WASI-I2C must have at least an independent implementation for the following platforms: Linux, ARM and RISC-V. Furthermore, an implementation also needs to be provided for a microcontroller. Here, a Cortex-M4 is targeted as the lowerbound.
+WASI-I2C must have at least an independent implementation for the following platforms: Linux, ARM, RISC-V and a RTOS (Zephyr or FreeRTOS). Furthermore, an implementation also needs to be provided for a microcontroller. Here, a Cortex-M4 is targeted as the lowerbound.
```

Comment:
I didn't hear the objections but the way the list is enumerated here is a bit confusing.  We have OSish things - Linux and RTOS and CPU Families ARM, and RISC- 2 different axis types.  Additionally mentioning Cortex M4 as a microcontroller - which itself is an Arm family MCU.

Is the intention here for this to say:

Linux (ARM & RISC-V), and a RTOS (Zephyr or FreeRTOS).  

this translates to 

Must implement on:

- [ ]  Linux (ARM)
- [ ] Linux (RISC-V)
- [ ] RTOS (Zephyr or FreeRTOS)
- [ ] Microcontroller Implementation (Cortex M4)

Reply:
Yes, sorry for the confusion. I changed it so that it's now an unordered list.